source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -559,10 +563,10 @@ namespace AdaptiveCards
                 AdaptiveColumnSetRenderer();
             };
 
-            [default_interface] runtimeclass AdaptiveFactRenderer : IAdaptiveElementRenderer
+          /*  [default_interface] runtimeclass AdaptiveFactRenderer : IAdaptiveElementRenderer
```

Comment:
I haven't encountered a place where it is used. It's only possible to render a **FactSet** according to https://adaptivecards.io/explorer/FactSet.html.
Although, documentation also mentions https://adaptivecards.io/explorer/Fact.html - it can only be part of a **FactSet** and cannot be used individually.
I'm quite certain we can get rid of this **FactRenderer** declaration in the IDL. (there's no implementation I believe)

Reply:
@paulcam206 might be able to help confirm this